### ISSUE TITLE
Unrecognized error message in Annotation mode

### ISSUE THREAD

**Issue by SJones-Appen** [webanno/webanno#686]
We've encountered a new error message in Annotation mode. The message is either

Error: conversion = %’ 

or 

Error: conversion = comma’

Sometimes, but not always, the errors make it impossible to annotate a span. Screenshot attached.

Thanks!

![error](https://user-images.githubusercontent.com/29804555/32080879-f5969876-ba66-11e7-99f6-b6e9df318b79.jpg)

**Comment by reckart** [comment]
Hm, this looks odd. I just tried annotating a text containing % signs in a recent WebAnno 3 SNAPSHOT build, which seems to be ok. Does the problematic text span contain a % sign? Which version are you using?

**Comment by SJones-Appen** [comment]
Hello, thanks for looking at this. V3.2.1, now in production. The error seems to affect only % signs and commas.

**Comment by reckart** [comment]
I think that I can reproduce the problem. It seems not to be related to the annotated text, but rather to value of a feature. I.e. if I create a Lemma annotation and set the value to `%`, then I see this error.

**Comment by reckart** [comment]
```
2017-11-03 11:01:52 ERROR [admin] AnnotationDetailEditorPanel - Error: Conversion = '%'
java.util.UnknownFormatConversionException: Conversion = '%'
	at java.util.Formatter.checkText(Formatter.java:2579) ~[?:1.8.0_131]
	at java.util.Formatter.parse(Formatter.java:2565) ~[?:1.8.0_131]
	at java.util.Formatter.format(Formatter.java:2501) ~[?:1.8.0_131]
	at java.util.Formatter.format(Formatter.java:2455) ~[?:1.8.0_131]
	at java.lang.String.format(String.java:2940) ~[?:1.8.0_131]
	at de.tudarmstadt.ukp.clarin.webanno.ui.annotation.detail.AnnotationDetailEditorPanel.writeFeatureEditorModelsToCas(AnnotationDetailEditorPanel.java:1893) ~[classes/:?]
	at de.tudarmstadt.ukp.clarin.webanno.ui.annotation.detail.AnnotationDetailEditorPanel.actionAnnotate(AnnotationDetailEditorPanel.java:1364) ~[classes/:?]
	at de.tudarmstadt.ukp.clarin.webanno.ui.annotation.detail.AnnotationDetailEditorPanel.actionAnnotate(AnnotationDetailEditorPanel.java:1221) [classes/:?]
	at de.tudarmstadt.ukp.clarin.webanno.ui.annotation.detail.AnnotationDetailEditorPanel$AnnotationFeatureForm$FeatureEditorPanelContent$2.onUpdate(AnnotationDetailEditorPanel.java:934) [classes/:?]
	at org.apache.wicket.ajax.form.AjaxFormComponentUpdatingBehavior.onEvent(AjaxFormComponentUpdatingBehavior.java:153) [wicket-core-6.26.0.jar:6.26.0]
...
```

**Comment by reckart** [comment]
I have just released WebAnno 3.2.4 which fixes this bug: https://github.com/webanno/webanno/releases/tag/webanno-3.2.4
